composer.json

```diff
@@ -14,9 +14,10 @@
     "type": "lib",
     "require": {
         "php": "^7.0",
-        "bestit/commercetools-odm": "dev-master",
```

Comment:
Whats with https://getcomposer.org/doc/04-schema.md#replace ?

Reply:
Lets discuss it tomorrow.